packages/types/src/documents/app/datasource.ts

```diff
@@ -12,7 +12,7 @@ export interface Datasource extends Document {
   }
   plus?: boolean
   entities?: {
-    [key: string]: Table
+    [key: string]: any
```

Comment:
Is there a difference between an entity and a table? If so, would be good to type whatever `Entity` is rather than `any`. Any thoughts here @mike12345567 ?